config.json

```diff
@@ -154,6 +154,23 @@
       "nodeUrls": [],
       "shortName": "Stacks",
       "type": "STX"
+    },
+    {
+      "coinApi": "evm",
+      "explorerUrl": "https://arbiscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Ethereum on Arbitrum",
```

Comment:
Both 'name' and 'shortName' as for display, are we sure 'ethereum on arbitrum' is the correct thing? I though it was called 'Arbitrum One'

Reply:
eg, AVAX name is 'Avalanche C-Chain' and shortName is 'Avalanche'

Reply:
@paulo-bc Arbitrum One is the network name (corresponding to relevant chainId), the asset is Ethereum

Reply:
Just got the latest comment - thank, will update

Reply:
I'm not super familiar on how this is used, but keep in mind Avalanche is not a L2, the native coin is AVAX while here the network name is Arbitrum One, the native coin is Ethereum (ARBETH for us), so if this is used to show as network only, I guess it is ok, now if this is used to show some coin name or something similar this is odd. 

Here is trustwallet definition:
https://github.com/trustwallet/assets/blob/a3823f660dd68e0fe99944e931142360cf06e360/blockchains/arbitrum/info/info.json 

I believe this is the first time we are having this, the native coin being not exactly the same as the network.

Reply:
Hey @mbratfisch-bc, I updated my comment, I think the issue raised because I said 'avax' and also got confused re 'Arbitrum One' (that is not what we are adding right)?

The Arbitrum network we are adding is a EVM right? (I see type is EVM)
If so, would make sense that its name and shortName to be just 'Arbitrum'.

Reply:
Also, `networkTicker` doesn't need to be the native asset ticker tho. Here we would use the market consensus (that I think is ARETH, but I am not sure).